test/sqllogictest/joins.slt

```diff
@@ -53,22 +53,33 @@ NULL  NULL  4  r4
 1     l1    1  r1
 3     l3    3  r3
 
-
 # Test that columns detected to be equivalent retain the names that the user
 # asks for. Protects against regression of #1217.
 query II colnames,rowsort
 SELECT ra, r.ra FROM l JOIN r ON l.la = r.ra LIMIT 0
```

Comment:
Perhaps we should add a version of the other "columns detected" tests where the equality is reversed too.